api/global/internal/meter.go

```diff
@@ -50,21 +50,22 @@ type meterProvider struct {
 	delegate metric.Provider
 
 	lock   sync.Mutex
-	meters map[string]*meter
+	meters map[string]*meterEntry
 }
 
-type meter struct {
-	delegate unsafe.Pointer // (*metric.Meter)
-
-	provider *meterProvider
-	name     string
+type meterImpl struct {
+	delegate unsafe.Pointer // (*metric.MeterImpl)
 
 	lock       sync.Mutex
-	registry   map[string]metric.InstrumentImpl
 	syncInsts  []*syncImpl
 	asyncInsts []*asyncImpl
 }
 
+type meterEntry struct {
+	registry metric.MeterImpl
```

Comment:
`registry` is an odd name. Would `unique` be okay instead?

Reply:
Done.